assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -169,11 +169,18 @@ function SingleSelectionMovable({ selectedElement, targetEl, pushEvent }) {
     !snapDisabled && (!isDragging || (isDragging && !activeDropTargetId));
   const hideHandles = (isDragging && isMaskable) || Boolean(draggingResource);
 
+  const minWidth = dataToEditorY(resizeRules.minWidth);
```

Comment:
`dataToEditorX` instead?

Reply:
Both functions are just `y = (y * pageHeight) / PAGE_HEIGHT)` with `pageHeight=PAGE_HEIGHT` so it's just `y`, but I will change that for better readability (and sacrifice a few bytes 😉).

Reply:
They both produce the same result but I changed it for clarity.